### PR TITLE
kubevirt: Update text in delete vmi users alert

### PR DESCRIPTION
**Pull request by yaacov**
fix data lose alert message for looged in users https://github.com/openshift/console/pull/5733#discussion_r443891837.

### CODE REVIEW

**Comment by yaacov** [comment]
@pcbailey please review

File: frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/vmi-users-alert.tsx [hunk 1]
```diff
@@ -38,8 +38,8 @@ export const VMIUsersAlert: React.FC<VMIUsersAlertProps> = ({
       <Button variant="link" isInline onClick={onLinkClick}>
         {pluralize(userListLength, 'User')}
       </Button>{' '}
-      currently logged in to this VM. Proceeding with this operation may cause logged in users data
-      loss.
+      currently logged in to this VM. Proceeding with this operation may cause logged in users to
+      lose data. loss.
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/vmi-users-alert.tsx:42]: Remove the `lose.` at the end

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/modals/menu-actions-modals/vmi-users-alert.tsx]: thanks !! fixed

**Comment by pcbailey** [comment]
Ido was nice enough to approve it already, but I'll stamp it, too. =)
/lgtm

**Pull request merged**